dpnp/backend/extensions/lapack/getrs.cpp

```diff
@@ -41,7 +41,7 @@ namespace mkl_lapack = oneapi::mkl::lapack;
 namespace py = pybind11;
 namespace type_utils = dpctl::tensor::type_utils;
 
-typedef sycl::event (*getrs_impl_fn_ptr_t)(sycl::queue,
+typedef sycl::event (*getrs_impl_fn_ptr_t)(sycl::queue &,
```

Comment:
Could you please update for other functions also, probably by separate PR?

Reply:
Sure. I will do it in the next PR.
Thanks.